deploy/role.yaml

```diff
@@ -21,6 +21,7 @@ rules:
   - pods
   - services
   - endpoints
+  - persistentvolumes
```

Comment:
Looks like we are very relaxed on RBAC ( All verbs on above objects . I dont think thats necessary) which is not a good practise in general.

Reply:
I won't change the RBAC for other objects, but I can implement a more restricted set of verbs for PVs specifically.

Reply:
Addressed.